components/panther.rst

```diff
@@ -0,0 +1,828 @@
+The Panther Component
```

Comment:
@alexandre-daubois thanks for importing the Panther docs 👏 

I have some concerns about the title and the location of this new article:

(1) About the **name**, I think that using a "marketing name" instead of a descriptive name was an strategic mistake here. I've talked many times about this internally to the Core Team, so I won't repeat it. But, the article title is super important for Google SEO and our own Meilisearch-based doc search. Nobody is going to look for "panther" because that's an animal and has nothing to do with Symfony, PHP or programming in general. I'd rename it to something like: "End-to-end Testing"

(2) About the **location**: we're trying to stop adding docs to `components/*` because we don't care much about standalone component docs. So, I wouldn't add new contents under `components/*`. Instead, we could move it e.g. to `testing/end_to_end.rst`

Reply:
Thank you Javier for your review, I addressed your comments :slightly_smiling_face: